openfisca_us/variables/gov/states/in/tax/income/exemptions/in_aged_low_agi_exemptions.py

```diff
@@ -0,0 +1,25 @@
+from openfisca_us.model_api import *
+
+
+class in_aged_low_agi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN base exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-1-3.5"  # (a)(5)(C)
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states["in"].tax.income.exemptions
+        filing_status = tax_unit("filing_status", period)
+        threshold = p.aged_low_agi.threshold[filing_status]
+        aged_low_agi_exemption = p.aged_low_agi.amount
+        federal_agi = tax_unit("adjusted_gross_income", period)
+        aged_head = tax_unit("aged_head", period)
+        aged_spouse = tax_unit("aged_spouse", period)
+        income_eligible = where(federal_agi < threshold, 1, 0)
```

Comment:
Multiplying by bools works. Also adding the comment as `<` is less typical than `<=` IME.
```suggestion
        # The law specifies "Less than".
        income_eligible = federal_agi < threshold
```